apps/vite-playground/src/WalletPicker.tsx

```diff
@@ -77,7 +78,10 @@ export const WalletPicker = ({ skClient, setWallet }: Props) => {
           const derivationPath = getDerivationPathFor({ chain: chains[0], index: 1 });
           return skClient.connectLedger(chains[0], derivationPath);
         }
-
+        case WalletOption.KEEPKEY: {
+          const derivationPath = getDerivationPathFor({ chain: chains[0], index: 0 });
+          return skClient.connectKeepKey(chains[0], derivationPath);
```

Comment:
Can't KeepKey handle multiple chains connect at once? We do stuff like that normally on UI -> connect 5 chains at once.